transaction-negative-tests/Ballerina.toml

```diff
@@ -1,25 +1,25 @@
 [package]
 org = "ballerinai"
```

Comment:
Can we check whether we have a `config` dependency here and in other Ballerina.toml files and remove only that part?